src/job/job.spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jest */
+
+import { omit } from "lodash";
+
+import config from "../_config";
+import { xo } from "../util";
+
+describe("job", () => {
+  describe(".create() :", () => {
+    it("creates a new job", async () => {
+      const jobId = await xo.createJob({
+        job: {
+          type: "call",
+          key: "snapshot",
```

Comment:
Please add `name`, `timeout` properties to the this test and create a specific test for the property `userId`.